ledger-service/http-json-oracle/src/it/scala/http/HttpServiceWithOracleIntTest.scala

```diff
@@ -17,3 +17,7 @@ class HttpServiceWithOracleIntTest
 
   override def wsConfig: Option[WebsocketConfig] = None
 }
+
+final class HttpServiceWithOracleIntTestCustomToken
+    extends HttpServiceWithOracleIntTest
+    with AbstractHttpServiceIntegrationTestFunsCustomToken
```

Comment:
Maybe rename `AbstractHttpServiceIntegrationTestFunsCustomToken` to `AbstractHttpServiceIntegrationTestCustomToken`